README.md

```diff
@@ -1,52 +1,129 @@
-# GovCMS Project Scaffolding
+# GovCMS SaaS project
 
-## Known Issues
+[![pipeline status](https://projects.govcms.gov.au/GovCMS/saas-scaffold/badges/master/pipeline.svg)](https://projects.govcms.gov.au/GovCMS/saas-scaffold/commits/master)
 
-* Currently (Nov 2018), all local projects utilise the same LOCALDEV_URL - we are working to fix that.
-* This repository is still a Work-in-Progress, and may be subject to slight alterations
-* This process only applies to the [7.x-3.x](https://github.com/govCMS/govcms/tree/7.x-3.x) branch of GovCMS
+## Requirements
 
-## Requirements and Preliminary Setup
+* [Docker](https://docs.docker.com/install/) - a container management system
+* [pygmy](https://docs.amazee.io/local_docker_development/pygmy.html#installation) (you might need `sudo` for this depending on your ruby configuration)
+* [Ahoy](http://ahoy-cli.readthedocs.io/en/latest/#installation) - a shortcut manager for saving your precious fingers
+* [Portainer](https://portainer.io/install.html) - not critical, but a nice web UI for managing Docker stuff
 
-* [Docker](https://docs.docker.com/install/) - Follow documentation at https://docs.amazee.io/local_docker_development/local_docker_development.html to configure local development environment.
 
-* [Mac/Linux](https://docs.amazee.io/local_docker_development/pygmy.html) - Make sure you don't have anything running on port 80 on the host machine (like a web server):
+## Known issues
+
+1. This process only applies to the `7.x-3.x` branch of GovCMS
+2. Currently (Nov 2018), all local projects utilise the same LOCALDEV_URL. The URL used is hardcoded in. GovCMS is aware and working on a fix. To access different sites, shut down the containers of all except the one you want to see at that URL.
+3. The container 'test' cannot have its name changed. This prevents Drupal from being able to connect to the database for some reason.
+4. When logging into the site for the first time, the 'Reset password' page does not allow resetting the password, complaining  `Password may only be changed in 24 hours from the last change`. See Step 5 of 'Spin up a vanilla govCMS site' for the workaround. 
 
-        gem install pygmy
-        pygmy up
 
-* [Windows](https://docs.amazee.io/local_docker_development/windows.html):    
 
-        git clone https://github.com/amazeeio/amazeeio-docker-windows amazeeio-docker-windows; cd amazeeio-docker-windows
-        docker-compose up -d; cd ..
+## Setup
 
-* [Ahoy (optional)](http://ahoy-cli.readthedocs.io/en/latest/#installation) - The commands are listed in `.ahoy.yml` all include their docker-compose versions for use on Windows, or on systems without Ahoy.
 
-## Project Setup
+### Spin up a vanilla govCMS site
+
+This lets you quickly whip up a site in Docker, but without persistent storage, i.e. **if you shut down your containers, you lose your changes**. 
```

Comment:
Dumb question, I'm guessing this also applies to leaving them running but shutting down your HOST machine?